### PR TITLE
docs: improve wording in rook command documentation

### PR DESCRIPTION
**Pull request by obnoxxx**
just some rewording that occurred to me when reading the docs

### CODE REVIEW

File: docs/rook.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Rook
 
-Rook is parent command which requires sub-command to get working. Currently, rook supports these sub-commands:
+Rook is a parent command and  requires sub-command to get working. Currently, rook supports the following  sub-commands:
```
Comment (travisn) [inline docs/rook.md:3]: How about simplifying?
```suggestion
The `rook` command supports the following  sub-commands:
```

File: docs/rook.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+1. `purge-osd <osd-id> [--force]` : [purge osd](#purge-osd) permanently remove an OSD from the cluster. Multiple OSDs can be removed in a single command with a comma-separated list of IDs.
 2. `version`: [version](#version) prints the rook version.
 3. `status`      : [status](#status) print the phase and conditions of the CephCluster CR
 4. `status all`  : [status all](#status-all) print the phase and conditions of all CRs
 5. `status <CR>` : [status  cr](#status-cr-name) print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc
 
-## Purge OSD
+## Purge OSDs
```
Comment (travisn) [inline docs/rook.md:11]: This section shows an example of purging a single OSD. How about keeping it singular?
```suggestion
## Purge an OSD
```

File: docs/rook.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+Permanently remove an OSD from the cluster. Multiple OSDs can be removed in one invocation with a comma-separated list of IDs.
```
Comment (travisn) [inline docs/rook.md:13]: How about moving the comment about multiple OSDs below the example? The example is only for a single OSD.
```suggestion
Permanently remove an OSD from the cluster. 

(below the output)
Multiple OSDs can be removed in one invocation with a comma-separated list of IDs.
```

File: docs/rook.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
```
Comment (travisn) [inline docs/rook.md:14]: Let's add a warning
```suggestion
!!! warning
    Data loss is possible when passing the --force flag if the PGs are not healthy on other OSDs.
```

File: docs/rook.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ kubectl rook-ceph rook status
 #       "lastHeartbeatTime": "2022-09-14T08:57:02Z",
 #       "lastTransitionTime": "2022-09-14T08:56:59Z",
 #       "message": "Cluster created successfully",
-#       "reason": "ClusterCreated",
+#       "reason": "ClusterCreated",`
```
Comment (travisn) [inline docs/rook.md:65]: revert?

**Review by travisn — CHANGES_REQUESTED** [review]

**Comment by obnoxxx** [comment]
@travisn, I think I addressed all your requests  and/or committed and squashed all your suggestions

File: docs/rook.md (hunk 5 above)
Comment (obnoxxx) [reply docs/rook.md]: done

**Review by travisn — CHANGES_REQUESTED** [review]

File: docs/rook.md [hunk 6]
```diff
@@ -1,17 +1,21 @@
...
 4. `status all`  : [status all](#status-all) print the phase and conditions of all CRs
 5. `status <CR>` : [status  cr](#status-cr-name) print the phase and conditions of CRs of a specific type, such as 'cephobjectstore', 'cephfilesystem', etc
 
-## Purge OSD
+## Purge an OSD
 
-Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+Permanently remove an OSD from the cluster. 
 
+Multiple OSDs can be removed in one invocation with a comma-separated list of IDs.
```
[the remark this answers is not in the document]
Comment (obnoxxx) [reply docs/rook.md]: done!

**Review by travisn — APPROVED** [review]

File: docs/rook.md [hunk 7]
```diff
@@ -1,17 +1,21 @@
...
 
-## Purge OSD
+## Purge an OSD
 
-Permanently remove an OSD from the cluster. Multiple OSDs can be removed with a comma-separated list of IDs.
+Permanently remove an OSD from the cluster. 
 
+
+!!! warning
+    Data loss is possible when passing the --force flag if the PGs are not healthy on other OSDs.
```
Comment (BlaineEXE) [inline docs/rook.md:17]: I'm not sure that all users will know what a PG is. It might be good to link to ceph docs.

**Review by BlaineEXE — APPROVED** [review]
Approving with a minor suggestion

File: docs/rook.md (hunk 7 above)
Comment (obnoxxx) [reply docs/rook.md:17]: good point, @BlaineEXE ! would you consider it sufficient to add that in a follow-up PR?

Comment (BlaineEXE) [reply docs/rook.md:17]: My experience is that most follow-up PRs don't actually happen. This is already merged, so it'll have to be in follow-up

**Pull request merged**